Automattic/vip-cli
# Add a basic SQL file checker

## Description

An initial pass of a utility to run basic string-based checks on an SQL import file.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-validate.js <filename.sql>`
